build.gradle.kts

```diff
@@ -18,6 +19,8 @@ repositories {
     mavenCentral()
 }
 
+extra["testcontainersVersion"] = "1.17.2"
```

Comment:
What is this for? Can you use the version variable defined earlier on the right hand side?